restructuredtext_lint/cli.py

```diff
@@ -1,46 +1,54 @@
 import argparse
 import json
 import sys
+import locale
 
 from .lint import lint_file
 
 
-def _main(filepath, format='text', stream=sys.stdout, encoding=None):
+def _main(filepaths, format, encoding, stream=sys.stdout):
```

Comment:
While I appreciate the notion of relocating defaults from `_main` to the CLI parser, it is creating noise in this PR. Please relocate those changes to a separate PR.